TMDBApp/TMDBApp/AppDelegate.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AppDelegate.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 17/12/2016.
+//  Copyright © 2016 Richard Leh. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        UIApplication.shared.statusBarStyle = UIStatusBarStyle.lightContent
+        
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {
```

Comment:
Hello Marcelo! Thank you for this opportunity =)
I'll answer your questions.

-
Nothing happens If I don't implemente these methods.
I could remove all them.